.github/workflows/terraform-compatibility-matrix.yml

```
@@ -50,6 +50,7 @@ jobs:
   
   slack-notification:
     needs: ["run-test-supported-versions"]
+    if: ${{ !cancelled() }}
```

Shouldn't this run in the event of failure as well?
```suggestion
    if: ${{ !cancelled() && failure() }}
```

---

Since this is a monthly run, we want to send the slack notification both when it is successful and fails. `!cancelled()` makes it run independently of the jobs in `needs` being successful or not, so it would always send the notification.
Adding `failure()` would send the notification only when it fails